src/utils/user-utils.ts

```diff
@@ -0,0 +1,16 @@
+import type { User } from "src/generated/homeLambdasClient/models/User";
+import config from "src/app/config";
+
+/**
+ * To get the severa user id when logged in, if severaUserId is not found, it will return the test user severa id
+ * 
+ * @param user 
+ * @returns 
+ */
+export const getSeveraUserId = (user: User | undefined): string => {
+	if (user?.attributes?.severaUserId) {
+    return user.attributes.severaUserId;
+  }
+  const testUserSeveraId = config.user.testUserSeveraId || "";
+  return testUserSeveraId;
+};
```

Comment:
Is the formatting alright in here? That "if" seems to be in the wrong place

Reply:
Simplified this a bit